apps/system/js/remote/message_controller.js

```diff
@@ -40,7 +46,8 @@
 
     _handle_message: function(evt) {
       var data = evt.data;
-      if (data.target !== this.displayId) {
+      if (parseInt(data.target) !== this.displayId &&
```

Comment:
According to [1], I would recommend you to always specify the `radix` of `parseInt`.

[1] https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#Parameters